p2p/src/config.rs

```diff
@@ -66,6 +66,8 @@ impl From<NodeType> for BTreeSet<PubSubTopic> {
 pub struct P2pConfig {
     /// Address to bind P2P to.
     pub bind_addresses: Vec<String>,
+    /// SOCKS5 proxy.
+    pub proxy: Option<String>,
```

Comment:
Please call this `socks5_proxy`... we may support other types of proxies in the future.

I see you called this just `proxy` in many please... please rename them all to `socks5_proxy` where it's exclusive for socks5 proxy.

Reply:
Sure, updated.